src/DateTimeValidator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace ipl\Validator;
+
+use DateTime;
+use ipl\I18n\Translation;
+
+/**
+ * Validator for date-and-time input controls
+ *
+ * @see \Icinga\Web\Form\Element\DateTimePicker For the date-and-time input control.
+ */
+class DateTimeValidator extends BaseValidator
+{
+    use Translation;
+
+    protected $local;
+
+    /**
+     * Create a new date-and-time input control validator
+     *
+     * @param bool $local
+     */
+    public function __construct($local)
+    {
+        $this->local = (bool) $local;
+    }
+
+    /**
+     * Is the date and time valid?
+     *
+     * @param   string|DateTime $value
+     * @param   mixed           $context
+     *
+     * @return  bool
+     */
+    public function isValid($value, $context = null)
+    {
+        // Multiple isValid() calls must not stack validation messages
+        $this->clearMessages();
+
+        if (! $value instanceof DateTime && ! is_string($value)) {
+            $this->addMessage(
+                $this->translate('Invalid type given. Instance of DateTime or date/time string expected')
+            );
+            return false;
+        }
+
+        if (! $value instanceof DateTime) {
+            $format = $baseFormat = $this->local === true ? 'Y-m-d\TH:i:s' : DateTime::RFC3339;
+            $dateTime = DateTime::createFromFormat($format, $value);
+
+            if ($dateTime === false) {
+                $format = substr($format, 0, strrpos($format, ':'));
+                $dateTime = DateTime::createFromFormat($format, $value);
+            }
```

Comment:
That comes from here: https://github.com/Icinga/icingaweb2/issues/1391

Have you checked if that's still the case? (Try Chrome and Safari) I highly doubt that, because otherwise we'd have noticed that already I suppose.